donkeycar/parts/tub_v2.py

```diff
@@ -109,9 +122,21 @@ class TubWriter(object):
     """
     A Donkey part, which can write records to the datastore.
     """
+
+    class TubRecordCached:
```

Comment:
I thought you were planning to get rid of this completely and use a tuple instead ?

Reply:
Yes, this should be removed entirely.